kratos/solving_strategies/strategies/solving_strategy.h

```diff
@@ -402,6 +404,26 @@ class SolvingStrategy
         KRATOS_CATCH("")
     }
 
+    /**
+     * @brief This method provides the defaults parameters to avoid conflicts between the different constructors
+     * @return The default parameters
+     */
+    virtual const Parameters GetDefaultParameters() const
+    {
+        const Parameters default_parameters = Parameters(R"(
+        {
+            "name"                         : "solving_strategy",
+            "move_mesh_flag"               : false,
+            "echo_level"                   : 1,
+            "build_level"                  : 2,
+            "builder_and_solver_settings"  : {},
+            "convergence_criteria_settings": {},
+            "linear_solver_settings"       : {},
+            "scheme_settings"              : {}
```

Comment:
```suggestion
            "echo_level"                   : 1,
            "build_level"                  : 2
```

those should come from derived classes